src/tallies/tally.cpp

```diff
@@ -1258,10 +1258,13 @@ extern "C" int openmc_tally_set_nuclides(
     } else {
       auto search = data::nuclide_map.find(word);
       if (search == data::nuclide_map.end()) {
-        set_errmsg("Nuclide \"" + word + "\" has not been loaded yet");
-        return OPENMC_E_DATA;
+        int err = openmc_load_nuclide(word.c_str(), nullptr, 0);
+        if (err < 0) {
+          set_errmsg(openmc_err_msg);
+          return OPENMC_E_DATA;
+        }
```

Comment:
I assume this change is so that we can load "missing" nuclides for the tally without needing the nuclide to be in the material. Did you forget to add it in the previous PRs?